client/src/Components/NavigationBar.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+
+import { makeStyles } from "@material-ui/core/styles";
+import AppBar from "@material-ui/core/AppBar";
+import Toolbar from "@material-ui/core/Toolbar";
+
+import Button from '@material-ui/core/Button';
+import Typography from "@material-ui/core/Typography";
+import DashboardRoundedIcon from '@material-ui/icons/DashboardRounded';
+import CalendarTodayOutlinedIcon from '@material-ui/icons/CalendarTodayOutlined';
+import AddIcon from '@material-ui/icons/Add';
+import AccountCircleIcon from '@material-ui/icons/AccountCircle';
+import IconButton from '@material-ui/core/IconButton';
+
+const useStyles = makeStyles(theme => ({
+    iconButtonLabel: {
+     display: 'flex',
+    },
+    appBar: {
+      background: "#FFFFFF",
+      boxShadow: "none",
+      display: 'flex',
+      flexDirection: "row",
+      alignItems: 'center',
+      justifyContent: 'space-between',
+    },
+    toolBar: {
+      justifyContent: 'center',
+      alignItems: 'center',
+      flexGrow: 2,
+    },
+    createAndProfileBar: {
+      justifyContent: 'space-between',
+    },
+    centerDiv: {
+      alignItems: 'center',
+    },
+    menuButton: {
+        color: 'black',
+    },
+    button: {
+      margin: theme.spacing(1),
+    },
+    title: {
+      color: 'black',
+    }
+  }));
+
+const NavigationBar = () => {
+    const classes = useStyles();
+    return(
+      <AppBar position="static" className={classes.appBar}>
+        <Typography variant="title">
+          <img src="/images/logo.png" alt="bug"  />
```

Comment:
You may want to move the logo section inside the toolbar, cause it does not have any padding outside it

Reply:
I ended up using another toolbar; I couldnt figure out how to add padding and adding this logo into the other toolbar causes a problem in that the other toolbar holds two elements at center and this logo needs to be at the far left. The current solution results in the same appearance tho as in the mock up.